src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -80,10 +80,11 @@ public string GetRootedPath(IExecutionContext context, string path)
             return rootedPath;
         }
 
-        // TODO: This method seems not relevant to Release Extension, refactor it
         public void ConvertLocalPath(IExecutionContext context, string localPath, out string repoName, out string sourcePath)
         {
-            throw new NotImplementedException();
+            Trace.Info($"Received localpath {localPath}");
+            repoName = string.Empty;
+            sourcePath = string.Empty;
         }
```

Comment:
Looks good. Pl. validate if this fixes the failing release & then we should be good to go.